stylesheets/_session_signin.scss

```diff
@@ -152,21 +152,21 @@
 
   &-terms-conditions-agreement {
     padding-top: $session-margin-md;
-    @include session-color-subtle(var(--color-text));
 
+    color: var(--color-text-subtle);
     text-align: center;
     font-size: 12px;
 
     a {
       white-space: nowrap;
       font-weight: bold;
       text-decoration: none;
-      @include session-color-subtle(var(--color-text));
 
+      color: rgba(var(--color-text), 0.6);
```

Comment:
I've fixed this in the theming branch but this probably doesn't work correctly? Based on what we talked about yesterday.

Reply:
forced it to black, as there is no theme on the registration yet

Reply:
👍